Ghidra/Processors/AARCH64/src/main/java/ghidra/app/util/bin/format/elf/relocation/AARCH64_ElfRelocationHandler.java

```diff
@@ -71,6 +71,8 @@ public RelocationResult relocate(ElfRelocationContext elfRelocationContext,
 
 		boolean is64bit = true;
 		
+		boolean overflowCheck = true; // *_NC type relocations specify "no overflow check"
```

Comment:
We have generally omit overflow checks .  While this may be an over-simplification, we defer to the original compiler/linker to have performed such checks.

Reply:
Roger, I can remove the overflow checks. Standby.